src/components/checkboxes/index.md.njk

```diff
@@ -0,0 +1,35 @@
+---
+title: Checkboxes
+description: Let users select one or more options by using the Checkboxes component.
+section: Components
+theme:
+status: Approved
+department: All
+aliases:
+copy: done
+layout: layout-pane.njk
+---
+
+Let users select one or more options by using the Checkboxes component.
+
+## When to use this component
+
+Use the Checkboxes component to help users:
+
+- select multiple options from a list
+- toggle a single option on or off
+
+## When not to use this component
+
+Don’t use the Checkboxes component if users can only choose one option from a selection. In this case, use the [Radios component](Insert URL to Radios component page).
+
+## How it works
+
+There are two ways to use the Checkboxes component. You can use HTML or, if you’re using [Nunjucks](https://mozilla.github.io/nunjucks/) or the [GOV.UK Prototype Kit](Insert URL), you can use the Checkboxes macro.
+
+[Live example with codeblock]
+
+## Research on this component
+
+Read a blog post about updates to the [Radios and Checkboxes components](https://designnotes.blog.gov.uk/2016/11/30/weve-updated-the-radios-and-checkboxes-on-gov-uk/).
```

Comment:
Missing opening `[` for the link.

Reply:
Ignore me, I can't read. However, ideally our links should be more descriptive than this – users of assistive technologies iterating through the links on the page will just see 'Radios and checkboxes components' which is not a good description for where it will take them /cc @amyhupe

Reply:
I agree. Can we put the opening [ before the word 'updates' please?

Reply:
Updated